packages/wdio-browserstack-service/src/service.ts

```diff
@@ -176,7 +176,7 @@ export default class BrowserstackService implements Services.ServiceInstance {
         await this._insightsHandler?.beforeHook(test, context)
     }
 
-    async afterHook (test: Frameworks.Test|CucumberHook, context: unknown, result: Frameworks.TestResult) {
+    async afterHook (test: Frameworks.Test|CucumberHook, context: unknown, hookName: string,  result: Frameworks.TestResult) {
```

Comment:
Shouldn't `hookName` come last?

Reply:
i've tried but i'm getting the `unused var` lint error, which is weird, since when the arg is in this location, the lint doesn't show the error
<img width="1102" alt="image" src="https://github.com/webdriverio/webdriverio/assets/48208030/7ecbeb31-fcc9-40ab-977c-40ec6f612090">

Reply:
The service doesn't need the parameter to be set, so you can just not add it there.

Reply:
I removed the parameter